>>> BaseGeometry = __import__("7-base_geometry").BaseGeometry
>>> bg = BaseGeometry()
>>> try:
...     bg.area()
... except Exception as e:
...     print(f"[Exception] {str(e)}")
...
[Exception] area() is not implemented
>>> try:
...     bg.integer_validator("me", 1)
... except Exception as e:
...     print(f"[Exception] {str(e)}")
...
>>> try:
...     bg.integer_validator("me", 0)
... except Exception as e:
...     print(f"[Exception] {str(e)}")
...
[Exception] me must be greater than 0
>>> try:
...     bg.integer_validator("me", -2)
... except Exception as e:
...     print(f"[Exception] {str(e)}")
...
[Exception] me must be greater than 0
>>> try:
...     bg.integer_validator("me", "go")
... except Exception as e:
...     print(f"[Exception] {str(e)}")
...
[Exception] me must be an integer
>>> try:
...     bg.integer_validator("me", 10.5)
... except Exception as e:
...     print(f"[Exception] {str(e)}")
...
[Exception] me must be an integer
>>> try:
...     bg.integer_validator("me", {3, 4})
... except Exception as e:
...     print(f"[Exception] {str(e)}")
...
[Exception] me must be an integer
>>> try:
...     bg.integer_validator("me", None)
... except Exception as e:
...     print(f"[Exception] {str(e)}")
...
[Exception] me must be an integer
>>> try:
...     bg.integer_validator("me", True)
... except Exception as e:
...     print(f"[Exception] {str(e)}")
...
[Exception] me must be an integer
>>> try:
...     bg.integer_validator("me", (4,))
... except Exception as e:
...     print(f"[Exception] {str(e)}")
...
[Exception] me must be an integer
>>> try:
...     bg.integer_validator("me", [5])
... except Exception as e:
...     print(f"[Exception] {str(e)}")
...
[Exception] me must be an integer
>>> try:
...     bg.integer_validator()
... except Exception as e:
...     print(f"[Exception] {str(e)}")
[Exception] integer_validator() missing 2 required positional arguments: 'name' and 'value'
>>> try:
...     bg.integer_validator("age")
... except Exception as e:
...     print(f"[Exception] {str(e)}")
[Exception] integer_validator() missing 1 required positional argument: 'value'
